evennia/contrib/worldloader/example_empty_world/worlddata/models.py

```diff
@@ -0,0 +1,121 @@
+from django.db import models
+
+
+#------------------------------------------------------------
+#
+# store all rooms
+#
+#------------------------------------------------------------
+class world_rooms(models.Model):
```

Comment:
I'm sorry if I'm nitpicking here, but why is this intermediary step necessary - that is, why is a separate database table needed holding only the text and not the actual objects? If uniqueness is (for some reason) required, why not just check this at creation-time? I see no documented use for these extra models, but maybe I'm not just seeing it.